roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,23 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
+
+from ansible.module_utils.docker_common import AnsibleDockerClient
+
+
+def main():
+    client = AnsibleDockerClient(
+        argument_spec=dict(
+            name=dict(type='list'),
```

Comment:
`name` doesn't appear to be used anywhere.

I would add `supports_check_mode=True` to params for AnsibleDockerClient.